chore: Announce CQ Marketplace (#15474)
cloudquery/cloudquery: website/pages/blog/announcing-cloudquery-marketplace.mdx

---
title: "Announcement: Selling on CloudQuery Hub (Marketplace)"
tag: announcement
date: 2023/11/27
description: "Enabling developers to sell on CloudQuery Hub"
author: yevgenypats
---

import { BlogHeader } from "../../components/BlogHeader"

<BlogHeader/>

Following the launch of [CloudQuery Hub](https://docs.cloudquery.io/blog/announcing-cloudquery-new-hub) last month and premium plugins,

> We are excited to announce that we are openning registration for early access for 3rd party developers who are interested in 

Reviewer: ```suggestion
We are excited to announce that we are opening registration for early access for 3rd party developers who are interested in 
```

> developing and selling plugins, transformations and visaulizations on CloudQuery Hub.

Reviewer: ```suggestion
developing and selling plugins, transformations and visualizations on CloudQuery Hub.
```

## Key Details

- Only Go plugins are currently supported for 3rd party developers to sell on CQ Hub (
  We will support eventually all other CQ SDK languages such as Python/Java/Javascript)
- Plugins are charged in a usage based pricing with $X per 1M rows per month that the developer sets.

> - Transformations and visualizations are charged as one time fee per 6 months (and will include support and updates for those 6 months).

Reviewer 2: ```suggestion
- Transformations and visualizations incur a one-time fee every six months, which includes support and updates for that period.
```

> - CloudQuery takes care of billing, usage and metering and charges 30% commission out of each sale.

Reviewer 2: ```suggestion
- CloudQuery handles billing, usage, and metering, and charges a 30% commission on each sale.
```

> - CloudQuery takes care of billing, charing, usage and metering and charges 30% comission out of each sale.

Reviewer: ```suggestion
- CloudQuery takes care of billing, usage and metering and charges 30% comission out of each sale.
```

> - **Early Registration discount**: Anyone who is registering till end of Q1 2024 will be able to redeem 15% discount on commission till end of 2024.

Reviewer 2: ```suggestion
- **Early Registration discount**: Those who register by the end of Q1 2024 can redeem a 15% discount on commission fees until the end of 2024.
```

> - **Early Registration discount**: Anyone who is registring till end of Q1 2024 will be able to redeem 15% discount on comission till end of 2024.

Reviewer: ```suggestion
- **Early Registration discount**: Anyone who is registering till end of Q1 2024 will be able to redeem 15% discount on commission till end of 2024.
```
